pkg/network/network.go

```diff
@@ -2,18 +2,18 @@ package network
 
 import (
 	"encoding/json"
-	"fmt"
 	"io"
 	"net/http"
-	"os"
 
+	"github.com/labstack/gommon/log"
```

Comment:
What about using massa-station log pkg instead ?

Reply:
I would require to initlialize it, so chose a logs file, I'll do it later

Reply:
https://github.com/massalabs/station-massa-wallet/issues/538